index.html

```diff
@@ -229,9 +338,9 @@ <h2 class="icons-header">LET'S CONNECT</h2>
           </ul>
         </div>
         <ul class="icons-lang">
-          <li><img src="./assets/images/js.png" alt="js" />Javascript</li>
-          <li><img src="./assets/images/html.png" alt="html" />HTML</li>
-          <li><img src="./assets/images/css.png" alt="Ellipse" />CSS</li>
+          <li class="li-icons"><img src="./assets/images/js.png" alt="js" />Javascript</li>
+          <li class="li-icons"><img src="./assets/images/html.png" alt="html" />HTML</li>
+          <li class="li-icons"><img src="./assets/images/css.png" alt="Ellipse" />CSS</li>
         </ul>
         <div>
           <ul class="lang">
```

Comment:
- [ ] The arrow in front of skills is not showing up. kindly consider writing the `src` at line 362 like this `src="./assets/images/arrow.png" alt="arrow.png"`